data/map systems.txt

```diff
@@ -3268,6 +3268,72 @@ system Albireo
 		distance 2211.99
 		period 1509.48
 
+
+system Alcor
+	pos -554 156
+	government Republic
+	attributes "bright star" "dirt belt" "notable star"
+	habitable 3208.255
+	arrival 3208.255
+	belt 1615 10
+	belt 1795 9
+	link Mizar
+	asteroids "medium rock" 6 2.064
+	asteroids "large rock" 1 1.602
+	asteroids "small metal" 2 2.741
+	asteroids "medium metal" 3 2.587
+	asteroids "large metal" 3 3.08
+	minables lead 3 4.053
+	trade Clothing 202
+	trade Electronics 797
+	trade Equipment 596
+	trade Food 294
+	trade "Heavy Metals" 907
+	trade Industrial 644
+	trade "Luxury Goods" 1211
+	trade Medical 727
+	trade Metal 475
+	trade Plastic 488
+	fleet "Small Southern Merchants" 2000
+	fleet "Small Southern Pirates" 3000
```

Comment:
```suggestion
	fleet "Small Southern Pirates" 3000
	fleet "Derelict Northern" 10000
```
Same fleets that are in Ipsing currently. More generally, I think you might have to also have a look at adding these derelict fleets in some of the new systems, since they were added to the game after this PR was opened.